### PR TITLE
AM2R: Auto Tracker templates

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
- Added: 2 templates for AM2R as 3-lines and 4-lines

To do:
1) add licensing in somehow. All 1.5.5 item sprites under CC BY SA 4.0
2) add credit to AbyssalCreature, Eskimode7, ShirtyScarab554

### CODE REVIEW

**Comment by JeffGainsNGames** [comment]
image
image

**Comment by JeffGainsNGames** [comment]
If I can get some direction on point 1) and 2) that would be very helpful to me as I am not familiar with licensing and credit like this.

**Comment by Miepee** [comment]
Not sure if necessary, but as detailed note:
The DNA sprite was made by abyssal, the morph ball and power grip sprite by shirtyscarab, the rest by eskimode

File: randovania/data/gui_assets/tracker/trackers.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
     "trackers": {
-        "prime1": {
+        "am2r": {
+            "Game Art (3 Lines)": "am2r-game-three-lines.json",
+			"Game Art (4 Lines)": "am2r-game-four-lines.json"
```
Comment (Miepee) [inline randovania/data/gui_assets/tracker/trackers.json:5]: This is formatted weirdly.

**Review by Miepee — COMMENTED** [review]
Personally would like if the names would be "7x3" / "5x4" instead of "3 lines" and "4 lines"

**Comment by JeffGainsNGames** [comment]
> Personally would like if the names would be "7x3" / "5x4" instead of "3 lines" and "4 lines"

The X-lines is consistent with the other tracker templates (not that I disagree with your proposal).
image

I'll make the change but just noting this.

File: randovania/data/gui_assets/tracker/game-images/am2r/licenses.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Assets by:
```
Comment (Miepee) [inline randovania/data/gui_assets/tracker/game-images/am2r/licenses.txt:1]: Maybe rename this file to `credits.txt` instead of `licenses.txt`

**Review by Miepee — APPROVED** [review]
Seems fine from what I can see, does work properly with debug connector too.

**Pull request merged**